.github/workflows/ci.yml

```diff
@@ -52,13 +52,37 @@ jobs:
         with:
           go-version: ${{ secrets.GO_VERSION }}
       - name: Run coverage
-        run: go test ./... -race -coverprofile=coverage.txt -covermode=atomic
+        run: go test --short ./... -race -coverprofile=coverage.txt -covermode=atomic
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v1
         with:
           files: ./coverage.txt
           flags: unittests
           fail_ci_if_error: true
+  integration-tests:
+    needs: [golangci]
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.1.0
+        with:
+          cluster_name: "kind-cluster"
+      - name: Run coverage
+        run: go test -run Integration ./... -race -coverprofile=coverage.txt -covermode=atomic
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          files: ./coverage.txt
+          flags: integrationtests
```

Comment:
Flag `integrationtests` already [exist](https://github.com/layer5io/meshery/blob/64c7f6a51807fd7c4603d63c00a0b4d06b91c858/.github/workflows/ci.yml#L183).

Some suggestions to the new name:
- backendintegrationtests
- gointegrationtests

If you have a better name it's ok. It could "anything" but different :)

Reply:
In Ash's voice, I choose "gointegrationtests".